src/main/java/it/gov/pagopa/payhub/activities/connector/organization/client/OrganizationSearchClient.java

```diff
@@ -24,5 +24,9 @@ public Organization findByOrgFiscalCode(String orgFiscalCode, String accessToken
         return organizationApisHolder.getOrganizationSearchControllerApi(accessToken)
                 .crudOrganizationsFindByOrgFiscalCode(orgFiscalCode);
     }
+    public Organization findById(String organizationId, String accessToken) {
```

Comment:
it's more correct to handle here the conversion beween Long and String for the field organizationId
ps: use String.valueOf instead toString